laundry-management-service/app/src/main/java/com/microservices/laundrymanagement/controllers/LaundryController.java

```diff
@@ -1,27 +1,43 @@
 package com.microservices.laundrymanagement.controllers;
 
 import com.microservices.laundrymanagement.dto.OrderSubmissionDto;
+import com.microservices.laundrymanagement.service.LaundryService;
 import com.microservices.laundrymanagement.service.OrderService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.*;
 
 @RestController
 @RequestMapping("laundry")
 public class LaundryController {
     private OrderService orderService;
+    private LaundryService laundryService;
+    private final Logger logger = LoggerFactory.getLogger(LaundryController.class);
```

Comment:
private final have precedence over private fields/ move them higher

Reply:
Hmmm... I used to think that Idea deals with such things:)